app/models/activity.py

```diff
@@ -26,15 +26,21 @@ def activity_versions_at(activities, at_time):
     )
 
 
+def is_activity_considered_work(activity_type):
+    return activity_type != ActivityType.TRANSFER
+
+
 class ActivityType(str, Enum):
     DRIVE = "drive"
     WORK = "work"
     SUPPORT = "support"
+    TRANSFER = "transfer"
     __description__ = """
 Enumération des valeurs suivantes.
 - "drive" : conduite du véhicule
 - "work" : travail sans déplacement du véhicule
 - "support" : accompagnement ou disponibilité
+- "transfer": temps de liaison
```

Comment:
Same ramrk as before, I would simply put
```suggestion
- "transfer": liaison d'un point à un autre
```